webview/cocoa.py

```diff
@@ -57,6 +57,16 @@ class BrowserDelegate(AppKit.NSObject):
         def webView_contextMenuItemsForElement_defaultMenuItems_(self, webview, element, defaultMenuItems):
             return nil
 
+        # Display a JavaScript alert panel containing the specified message
+        def webView_runJavaScriptAlertPanelWithMessage_initiatedByFrame_\
+                (self, webview, message, frame):
+                    AppKit.NSRunningApplication.currentApplication().activateWithOptions_(AppKit.NSApplicationActivateIgnoringOtherApps)
+                    alert = AppKit.NSAlert.alloc().init()
+
+                    alert.setMessageText_(localization["cocoa.alert"])
```

Comment:
The default window.alert in OSX has no title text, so this can be omitted or left empty